packages/documentation/pages/usage/components/form.vue

```diff
@@ -78,6 +78,12 @@
 					<h2>Personal Details</h2>
 					<KtFormControllerObject formKey="personalDetails">
 						<template #default="{ setValues, values: formObjectValues }">
+							<KtFieldNumber
+								formKey="age"
+								helpText="YEARS"
+								label="age"
+								placeholder="quel age as tu?"
```

Comment:
I'd just fix this to English be compliant with the rest of our docs 

we don't really need helptext, keep the label, and change the placeholder to `how old are you?`